index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+            <nav>
+                <ul class="header">
+                    <li class="time">9:41</li>
+                    <li class="cellular"><img src="./image/cellular.png" alt=""></li>
+                    <li class="wifi"><img src="./image/Wifi.png" alt=""></li>
+                    <li class="battery"><img src="./image/battery.png" alt=""></li>
+                   <li class="name">Paul Orife</li>
```

Comment:
- [ ] The use of this `logo name` is to help the user navigate back to the home page when he or she clicks on it, you can understand why it is required for it to be wrapped in an `a` tag instead, kindly fix this